### ISSUE TITLE
Compilation error: temporary value dropped while borrowed

### ISSUE THREAD

**Issue by vigress8** [jelmer/deb822-lossless#63]
```
Checking debian-control v0.1.17
error[E0716]: temporary value dropped while borrowed
   --> <...>/debian-control-0.1.17/src/changes.rs:152:14
    |
149 |         let subdir = if source.starts_with("lib") {
    |             ------ borrow later stored here
...
152 |             &source.chars().next().unwrap().to_string()
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ creates a temporary value which is freed while still in use
153 |         };
    |         - temporary value is freed at the end of this statement
    |
    = note: consider using a `let` binding to create a longer lived value

For more information about this error, try `rustc --explain E0716`.
error: could not compile `debian-control` (lib) due to 1 previous error
```

**Comment by jelmer** [comment]
What version of rustc are you using? This compiles fine for me on 1.80.

**Comment by vigress8** [comment]
Ah, I'm on
```
rustc 1.78.0 (9b00956e5 2024-04-29) (built from a source tarball)
```

**Comment by vigress8** [comment]
That fixed it, thanks!

**Comment by jelmer** [comment]
Thanks for confirming! I've released v0.1.18 with the fix.
